schema-guide.md

```diff
@@ -444,37 +486,49 @@ primary principle, "Importance", when others cite this work.
 
 ## Definitions
 
-TODO explain why we use definitions
+Some values in CFF files are valid in different fields.
+For example, `repository-code`, `url` and `license-url` all take URLs as values.
+
+The schema therefore has [*definitions*](https://json-schema.org/understanding-json-schema/structuring.html#definitions)
+of smaller subschemas, that can be reused in the schema from the respective field, 
+instead of having to duplicate them.
+For example, there is one definition for a valid URL value ([`definitions.url`](#definitionsurl)),
+that is being referenced from `repository-code`, `url` and `license-url`.
+
+**Note:** Definitions are NOT field names, although they may be called similarly.
+Do NOT use definitions or subkeys (like `definitions.alias` or `definitions.entity.alias`)
+when writing a CFF file.  
```

Comment:
The tone of voice here is bordering on the aggressive :) Maybe something like 

```suggestion
`definitions` and its subkeys should not be used as fields in `CITATION.cff` files.
```

Reply:
Additionally, we could append a correct usage example and an incorrect usage example to the Note paragraph.


```yaml
# incorrect
authors: 
  - definitions.alias: sdrukat
```
```yaml
# incorrect
authors: 
  - definitions: 
      alias: sdrukat
```
```yaml
# correct
authors: 
  - alias: sdrukat
```

Reply:
I see what you mean :slightly_smiling_face:. Will change accordingly.

Reply:
Done in 72c777c91c65d1db86685cec87eaadceb16b5b44